androidbrowserhelper/src/main/java/com/google/androidbrowserhelper/trusted/ChromeLegacyUtils.java

```diff
@@ -143,8 +144,12 @@ private static boolean checkChromeVersion(PackageManager pm, String packageName,
         return getVersionCode(pm, packageName) >= version;
     }
 
+    @SuppressWarnings("deprecation")
     static int getVersionCode(PackageManager pm, String packageName) {
         try {
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) {
+                return (int) pm.getPackageInfo(packageName, 0).getLongVersionCode();
```

Comment:
Could you add a comment here saying why this works and that we only guarantee it works with Chrome?

Reply:
From the Javadoc: `Return versionCode and versionCodeMajor combined together as a single long value. The versionCodeMajor is placed in the upper 32 bits.` - It seems this should work with any Android App, as the lower 32 bits are effectively the versionCode?

Reply:
Ah, fair enough then.